fladle-plugin/src/main/java/com/osacky/flank/gradle/FulladlePlugin.kt

```diff
@@ -138,3 +45,127 @@ class FulladlePlugin : Plugin<Project> {
     }
   }
 }
+
+fun configureLibraryModule(project: Project, flankGradleExtension: FlankGradleExtension) = project.run {
+  pluginManager.withPlugin("com.android.library") {
+    val fulladleModuleExtension = extensions.getByType(FulladleModuleExtension::class.java)
+    if (fulladleModuleExtension.enabled.get()) {
```

Comment:
the other application module code uses `if (!enabled)  return` can we make it consistent in both cases?

Reply:
Sure